freedomofpress/securedrop-client
# create prod-like dev env

# Description

Fixes issues where the gui looks different between prod and dev by supporting a dev env that uses system site-packages that you have to install ahead of time (see https://github.com/freedomofpress/securedrop-debian-packaging/blob/main/securedrop-client/debian/control).

# Test Plan

0. install these packages on your debian machine (this can be a vm in qubes or native install): https://github.com/freedomofpress/securedrop-debian-packaging/blob/main/securedrop-client/debian/control
1. `make venv-debian && source .venv-debian/bin/activate`
2. run the client (`./run.sh`) against a dev server
- [ ] Confirm you can see https://github.com/freedomofpress/securedrop-client/issues/1286

# Checklist

If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:


If these changes add or remove files other than client code, the AppArmor profile may need to be updated. Please check as applicable:


If these changes modify the database schema, you should include a database migration. Please check as applicable:
